README.md

```diff
@@ -120,6 +120,27 @@ No native dependencies.
 
 Build with `cargo` and use `ldd` to check that the resulting binary does not depend on OpenSSL anymore.
 
+#### Linux and Docker
+
+Also you can build the repo with 
+[Parity CI Docker image](https://github.com/paritytech/scripts/tree/master/dockerfiles/cachepot-ci):
+
+```bash
+docker pull paritytech/cachepot-ci:staging
```

Comment:
Could we replace this by `podman`? It works on at least linux and mac os oob.

Reply:
Yes, I can, but on Mac the `podman run ... -v ...` command won't work because it still has issues with volume mounting

Reply:
On the other hand, the topic is "Linux and Docker", so I think I'll change it to Podman. In this case it resolves the issue with different UID of host user and container user.

Reply:
Replaced with `podman`